lib/ansible/modules/cloud/docker/docker_swarm_info.py

```diff
@@ -119,6 +124,11 @@
       name: mynode
   register: result
 
+- name: Get the swarm unlock key
+  docker_swarm_info:
+    unlock_key: yes
+  register: result
+
 - debug:
```

Comment:
I think you should move this `debug` up before the example, and add a new `debug` here which outputs `result.swarm_unlock_key`.